pcap-new.c

```diff
@@ -44,6 +44,10 @@
 #include <stdlib.h>		// for malloc(), free(), ...
 #include <string.h>		// for strstr, etc
 
+#if defined(_MSC_VER) && !defined(strdup)
```

Comment:
Again, `portability.h` should handle this; why is this necessary?

Reply:
Same reason: corrects the warning.

Reply:
Again, why must this be handled in several places in the source rather than by a change to `portability.h`?